src/jit/codegencommon.cpp

```diff
@@ -10853,6 +10853,139 @@ unsigned CodeGen::getFirstArgWithStackSlot()
 
 #endif // !LEGACY_BACKEND && (_TARGET_XARCH_ || _TARGET_ARM64_)
 
+void CodeGen::genSinglePush()
+{
+    genStackLevel += sizeof(void*);
+}
+
+void CodeGen::genSinglePop()
+{
+    genStackLevel -= sizeof(void*);
+}
+
+/*****************************************************************************
+ *
+ *  Push the given registers.
+ *  This function does not check if the register is marked as used, etc.
+ */
+
+regMaskTP CodeGen::genPushRegs(regMaskTP regs, regMaskTP* byrefRegs, regMaskTP* noRefRegs)
```

Comment:
Since you are at it, please see if you could add standard function header for this one and genPopRegs() below.